### PR TITLE
New RDT: Better support custom hooks

### PR DESCRIPTION
**Pull request by bvaughn**
Builds on top of #9807; adds better support for custom hooks by showing original names and jump-to-definition links when possible.

### CODE REVIEW

**Comment by bvaughn** [comment]
Failed e2e tests seem related to hit count loading delays (maybe related to recent BAC changes?) and not this PR.

**Review by markerikson — APPROVED** [review]

**Comment by bvaughn** [comment]
Thanks for the review! I'm going to hold off and merge this directly into main once #9807 has been approved.

**Pull request merged**
